src/python/foglamp/data_purge/README.txt

```diff
@@ -0,0 +1,36 @@
+General Description: FOGL-200 is the responsibility for the purge process, which seats somewhere
```

Comment:
README should not exists in official release. Currently in branch only as a txt file with information regarding written code.